ui/components/image_generation.py

```diff
@@ -0,0 +1,7 @@
+import openai
+import streamlit as st
+
+def generate_image(text_input):
+    img_response = openai.Image.create(prompt=f"{text_input} a cartoon image", n=3, size="256x256")
```

Comment:
do we feel cartoon is the best image style here? or would it be better to do something like "photorealistic futuristic image"